src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -49,6 +50,15 @@
     private static final String ISO_8601 = "yyyy-MM-dd'T'HH:mm:ss";
     private static final String ISO_8601_WITH_TZ = "yyyy-MM-dd'T'HH:mm:ssX";
 
+    // Nearly ISO dates generated by git whatchanged --format=+ci
+    // Look like '2015-09-30 08:21:24 -0600'
+    // ISO is    '2015-09-30T08:21:24-06:00'
+    private static final Pattern NEARLY_ISO_DATE = Pattern.compile("^([0-9]{4}-[0-9]{2}-[0-9]{2})" + // YYYY-MM-DD
```

Comment:
I am not understanding your feedback.  Can you help me better understand?

I think you're suggesting that I should look for a parser in the joda-time package which will directly read the git output that is "nearly ISO 8601", rather than detecting the "nearly ISO 8601" with a regular expression and then rewriting the string to be an ISO 8601 form which the Java SimpleDateFormat can parse successfully.

 I think the joda-time implementation of [DateTime.parse(String str)](http://joda-time.sourceforge.net/apidocs/org/joda/time/format/ISODateTimeFormat.html#dateTime()) is worth an attempt to use it instead of the regex based reformatting.

Is that what you were suggesting, or were you recommending something completely different?